compose/ui/ui/build.gradle

```diff
@@ -174,6 +174,7 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
             skikoMain {
                 dependsOn(commonMain)
                 dependencies {
+                    implementation("org.jetbrains.kotlinx:kotlinx-datetime:0.4.0")
```

Comment:
Version 0.4.0 means that backward compatibility may break.
If our user uses another version of kotlinx-datetime in Desktop app (For example, 0.1.0).
Then version 0.4.0 will be forced to him/her.

I think a good way to create expect/actual fun getTimeInMilliseconds():Long
For **desktop** target use System.nanoTime() / 1_000_000L (**without** a datetime library),
For **jsNativeMain** targets use Clock.System.now().toEpochMilliseconds() (**with** a datetime library).

Reply:
Thanks!
I realized that we have `fun currentNanoTime()` available in skiko common. So I'll use it instead.